Colours_And_Fonts_index_v1.html

```diff
@@ -88,25 +86,57 @@ <h2>Information</h2>
+
+			<br>It is essential that all students on the trip play their part in risk management. Their personal safety is very much their responsibility, and they must always act in a manner that does not endanger either themselves or others. <br>It is important that they follow the instructions of 			 the staff at all times.<br>
 
-			<br>It is essential that all students on the trip play their part in risk management. Their personal safety is very much their responsibility, and they must always act in a manner that does not endanger either themselves or others. It is important that they follow the instructions of 			 the staff at all times.<br>
+			<br>All these activities are compulsory but have been designed to be both educational and fun. Students will be supervised at all times by Tawa College staff. If you have not already returned the permission forms given out earlier this term, <br>then please collect another copy of the forms               from the school office and hand them in to the student office along with $75.00 before Friday, October 25. If you have already returned the permission forms, then you only need to pay $75.00 before Friday, October 25.<br></p></div>
+				  
 
-			<br>All these activities are compulsory but have been designed to be both educational and fun. Students will be supervised at all times by Tawa College staff. If you have not already returned the permission forms given out earlier this term, then please collect another copy of the forms               from the school office and hand them in to the student office along with $75.00 before Friday, October 25. If you have already returned the permission forms, then you only need to pay $75.00 before Friday, October 25.<br></p></div>	
+	</div>
+			
+				
 			
 					</div>	
 			  <button onclick="topFunction()" id="myBtn" title="Go to top">TOP</button>
-
-			
-		</div>
-			<div class="footer">copyright &copy; Lynne Farah 2024, all rights reserved.</div>
-		</div>
 			
 		</div>
 		
+		<div class="box3">	
+			
+        <h3>Quick Links</h3>
+
+			
+			
 			
+		<h4>Contact US</h4><br></div>
+			
+  <div class="box4">
+	  
+	  	  		<p class="para2">
+  <a href="https://drive.google.com/drive/folders/1Pg_Zai_ZdXACZyPGJAtzp55o5cG0S4qQ" target="_blank" rel="noopener noreferrer">
+    Forms
+  </a>
+</p>
+
+	  
+			 <p class="para3">
+			<span class="Phone_number"><a href="tel:+6442328184">+64 4 232 8184</a></span>
+			<span class="Gmail"><a href="mailto:secretary@tawacollege.school.nz">secretary@tawacollege.school.nz</a></span>
+		    <span class="School_Address"><br>Duncan Street,Tawa,Wellington<br>5028</span></p>
+	  
+

```

Comment:
I added a box4 right above the footer so I can add quick links and contact us for end users to contact the school and get the information they need.